### ISSUE TITLE
Facial Cyanosis

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7124]
**Preferred term label:**
facial cyanosis

**Synonyms**
cyanotic face

_layperson term:_
bluish face

**Definition (free text, please give PubMed ID)**
Persistent blue discoloration of regions of the face.

**Parent term (use hpo.jax.org/app)**
HP:0001063 - Acrocyanosis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Comment by pnrobinson** [comment]
added with definition "Bluish discoloration of one or more regions of the face."
Also, I do not think this is "Acro"-cyanosis, see comment for other issue.
The new terms should be available with the next release in October.
